tools/cockpit.spec

```diff
@@ -144,6 +144,10 @@ install -p -m 644 tools/cockpit.pam $RPM_BUILD_ROOT%{_sysconfdir}/pam.d/cockpit
 rm -f %{buildroot}/%{_libdir}/cockpit/*.so
 install -p -m 644 AUTHORS COPYING README.md %{buildroot}%{_docdir}/%{name}/
 
+# Debian/Ubuntu logos are not available on rpm based distros
+rm -rf %{buildroot}/%{_datadir}/%{name}/branding/debian
+rm -rf %{buildroot}/%{_datadir}/%{name}/branding/ubuntu
```

Comment:
This is used in ```atomic run``` style containers, or when used with the docker run command that enables. We shouldn't remove the  branding from the RPMs.